viewer/packages/cad-geometry-loaders/src/sector/culling/OrderSectorsByVisibilityCoverage.ts

```diff
@@ -206,7 +206,8 @@ export class GpuOrderSectorsByVisibilityCoverage implements OrderSectorsByVisibi
         if (container === undefined) {
           throw new Error(`Model ${toBeFiltered.modelIdentifier} is not registered`);
         }
-        const isCameraInsideSector = isWithinSectorBounds(container.model, toBeFiltered.metadata, camera.position);
+        const metadata = toBeFiltered.metadata as BaseSectorMetadata & V8SectorMetadata;
```

Comment:
We should probably add a check here that the type is actually fulfilled, to avoid passing V9 stuff here.